src/main/kotlin/io/delilaheve/util/ConfigOptions.kt

```diff
@@ -269,12 +271,22 @@ object ConfigOptions {
                 .getBoolean(ALLOW_HEXADECIMAL_COLOR, DEFAULT_ALLOW_HEXADECIMAL_COLOR)
         }
 
+    /**
+     * Allow usage of minimessage formating
+     */
+    val allowMinimessage: Boolean
+        get() {
+            return ConfigHolder.DEFAULT_CONFIG
+                .config
+                .getBoolean(ALLOW_MINIMESSAGE, DEFAULT_ALLOW_MINIMESSAGE)
+        }
+
     /**
      * If one of the color component is enabled
      */
     val renameColorPossible: Boolean
         get() {
-            return allowColorCode || allowHexadecimalColor
+            return allowColorCode || allowHexadecimalColor || allowHexadecimalColor
```

Comment:
```suggestion
            return allowColorCode || allowHexadecimalColor || allowMinimessage
```
I assume you want to set allowMinimessage on the third OR statement?

Reply:
Oh yeah actually nice catch !